src/Core/Form/ChoiceProvider/DeliveryTimeNoteTypesProvider.php

```diff
@@ -42,24 +44,74 @@ final class DeliveryTimeNoteTypesProvider implements FormChoiceProviderInterface
      */
     private $translator;
 
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+
+    /**
+     * @var ConfigurationInterface
+     */
+    private $configuration;
+
+    /**
+     * @var int
+     */
+    private $contextLanguageId;
+
     /**
      * @param TranslatorInterface $translator
+     * @param RouterInterface $router
+     * @param ConfigurationInterface $configuration
+     * @param int $contextLanguageId
      */
     public function __construct(
-        TranslatorInterface $translator
+        TranslatorInterface $translator,
+        RouterInterface $router,
+        ConfigurationInterface $configuration,
+        int $contextLanguageId
     ) {
         $this->translator = $translator;
+        $this->router = $router;
+        $this->configuration = $configuration;
+        $this->contextLanguageId = $contextLanguageId;
     }
 
     /**
      * {@inheritDoc}
      */
     public function getChoices()
     {
+        $linkOpeningTag = sprintf(
+            '&nbsp;<a target="_blank" href="%s"><i class="material-icons">open_in_new</i>',
```

Comment:
looks abit weird having html here, but I suppose we don't have any other choice?

Reply:
but what would be the ideal scenario?
A separate choice type with dedicated html in theme for such cases?

Reply:
Yeah I guess it could be done with a special form theme that would override the option rendering when it detects it matches a certain value I agree this one is not perfect but it does the job I'd say It's not like it adds that many logic or markup
And we have many other cases where we include some HTML inside some wordings so I'd say it's acceptable